CONTRIBUTING.md

```diff
@@ -16,6 +16,28 @@ npm run lint  # runs the linter
 ```
 
 
+## Running the Tests
+
+There's a small amount of configuration needed to run the tests locally.
+Observe the bottom of the `.gitignore` file, where it ignores the
```

Comment:
Can you add a hyperlink to this file?

[`.gitignore`](https://github.com/hammer-io/tyr/blob/master/.gitignore)